src/frontend/staff-portal/src/app/components/contact-info/contact-info.component.ts

```diff
@@ -260,9 +260,9 @@ export class ContactInfoComponent implements OnInit {
 
       // set violation date and time
       let tempViolationDate = new Date(this.lastUpdatedDispute.issuedDate);
+      tempViolationDate.setHours(tempViolationDate.getHours() + 7);
```

Comment:
You cant just add 7 hours. That will not work. We have time zones and daylight savings.